corehq/ex-submodules/couchforms/models.py

```diff
@@ -146,13 +145,25 @@ def get(cls, docid, rev=None, db=None, dynamic_properties=True):
         # on cloudant don't get the doc back until all nodes agree
         # on the copy, to avoid race conditions
         extras = get_safe_read_kwargs()
-        return db.get(docid, rev=rev, wrapper=cls.wrap, **extras)
+        try:
+            return db.get(docid, rev=rev, wrapper=cls.wrap, **extras)
+        except ResourceNotFound:
+            from corehq.form_processor.exceptions import XFormNotFound
+            raise XFormNotFound
 
     @property
     def type(self):
         return self.form.get(const.TAG_TYPE, "")
         
     @property
+    def id(self):
+        return self._id
+
+    @property
+    def form_id(self):
+        return self._id
```

Comment:
well if we don't have the uuid as a primary key on the django model, we need another way to access the id since `django_model.id` won't get you the right thing. but yeah, this should be one or the other not both